packages/react-sdk/src/remix.ts

```diff
@@ -0,0 +1,47 @@
+const getRemixSegment = (segment: string) => {
+  if (segment === "*") {
+    return "$";
+  }
+  const match = segment.match(/^:(?<name>\w+)(?<modifier>\*|\?)?$/);
+  const name = match?.groups?.name;
+  const modifier = match?.groups?.modifier;
+  if (name) {
+    if (modifier === "*") {
+      return "$";
```

Comment:
imo should throw here

Reply:
Probably work in remix `[partofroute]:param` ?

Reply:
It does not. Why should it throw? This is "*" named equivalent. See getRemixParams set the name parsed from url pattern.